Replicator/tests/ReplicatorCollectionSGTest.cc

```diff
@@ -724,6 +724,74 @@ TEST_CASE_METHOD(ReplicatorCollectionSGTest, "Resolve Conflict SG", "[.SyncServe
     }
 }
 
+TEST_CASE_METHOD(ReplicatorCollectionSGTest, "Update Once-Conflicted Doc - SGColl", "[.SyncServerCollection]") {
+
+    _authHeader = "Basic c2d1c2VyOnBhc3N3b3Jk"_sl;
+    const string docID = timePrefix() + "uocd-doc";
+
+    // Create a conflicted doc on SG, and resolve the conflict
+    sendRemoteRequest("PUT", docID + "?new_edits=false",
+                      "{\"_rev\":\"1-aaaa\",\"scopes\":{\"flowers\":{\"collections\":{\"roses\":{}}}},\"foo\":1}"_sl, true);
+    sendRemoteRequest("PUT", docID + "?new_edits=false",
```

Comment:
Don't know whether
     `  \"scopes\":{\"flowers\":{\"collections\":{\"roses\":{}}}},`
has any special meaning to SG. Does it?

Reply:
I doubt it.  That JSON format is used when creating the DB endpoint in Sync Gateway.  It has no meaning in a document put.

Reply:
Going to change this with the PR Jianmin has merged for a sendRemoteRequest overload that includes collection parameter